### PR TITLE
docs: alternative installations

### LINKED ISSUE #227: docs: document installation on MacOS/Linux
- [x] add reference to `swissdatasciencecenter/renku` tap
- [x] `brew install renku`
- [ ] include installation process in main renku documentation
- [ ] use snap https://launchpad.net/~swissdatasciencecenter/+snap/renku/

### PR DESCRIPTION
**Pull request by jirikuncar**
(closes #227)

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -65,6 +65,47 @@ The latest code can be installed directly from the Git repository:
 
     $ pip install -e git+https://github.com/SwissDataScienceCenter/renku-python.git#egg=renku
 
+Use following installation steps based on your operating system and preferences
+if you would like to work only with the command line interface.
```
Comment (rokroskar) [inline README.rst]: Why "only"? As opposed to develop renku?

File: README.rst [hunk 2]
```diff
@@ -65,6 +65,47 @@ The latest code can be installed directly from the Git repository:
...
+
+The recommended way of installing Renku on MacOS is via `Homebrew <brew.sh>`_.
+
+::
+
+    $ brew tap swissdatasciencecenter/renku
+    $ brew install renku
+
+..
+  TODO: uncomment one the snap build works.
```
Comment (rokroskar) [inline README.rst]: Remove?

File: README.rst (hunk 1 above)
Comment (jirikuncar) [reply README.rst]: Just wanted to make sure they don't expect the whole platform to be installed.

File: README.rst (hunk 2 above)
Comment (jirikuncar) [reply README.rst]: I think we can keep it so when the #235 is merged we simply uncomment it.

File: README.rst [hunk 3]
```diff
@@ -65,6 +65,48 @@ The latest code can be installed directly from the Git repository:
...
+
+  Linux
+  ~~~~~
+
+  There are universal *snap* packages available for Renku. Make sure you
+  have ``snap`` command available or `install snapd
```
Comment (ciyer) [inline README.rst]: Need an article before snap:

    Make sure you have the ``snap`` command...

File: README.rst [hunk 4]
```diff
@@ -65,6 +65,48 @@ The latest code can be installed directly from the Git repository:
...
+
+      $ open https://launchpad.net/~swissdatasciencecenter/+snap/renku/
+      $ snap install renku
+
+Pip Script Installer (``pipsi``)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can use `pipsi <https://github.com/mitsuhiko/pipsi>`_ to isolate
+dependencies and to guarantee that there are no version conflicts. Make sure
+you have ``pipsi`` command correctly installed and ``~/.local/bin`` is in your
```
Comment (ciyer) [inline README.rst]: See above.

    Make sure you have the ``pipsi`` command...

**Pull request merged**